jin-sik/1254-deepest-leaves-sum/1254-deepest-leaves-sum.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Definition for a binary tree node.
+ * function TreeNode(val, left, right) {
+ *     this.val = (val===undefined ? 0 : val)
+ *     this.left = (left===undefined ? null : left)
+ *     this.right = (right===undefined ? null : right)
+ * }
+ */
+/**
+ * @param {TreeNode} root
+ * @return {number}
+ */
+var deepestLeavesSum = function(root) {
+    let sums = []
```

Comment:
```suggestion
    const sums = []
```

JS 변수의 원시타입, 참조타입 관련된 차이가 있어 const으로 선언하는게 좋아보입니다.